charmhelpers/contrib/templating/contexts.py

```diff
@@ -118,7 +118,8 @@ def juju_state_to_yaml(yaml_path, namespace_separator=':',
 
     if os.path.exists(yaml_path):
         with open(yaml_path, "r") as existing_vars_file:
-            existing_vars = yaml.load(existing_vars_file.read())
+            existing_vars = yaml.load(existing_vars_file.read(),
+                                      Loader=yaml.FullLoader)
```

Comment:
We should probably use `yaml.safe_load()` as we don't actually need the full loader capabilities.